flang/include/flang/Optimizer/Builder/Runtime/Command.h

```diff
@@ -0,0 +1,27 @@
+//===-- Command.h - generate assignment runtime API calls --------*- C++ -*-===//
```

Comment:
Nit: this is more than 80 characters (too many ---) and `generate assignment runtime` is probably not the right description here (maybe  `generate command line runtime`).

Reply:
Done.